app/assets/javascripts/modules/users/components/application-tokens/panel.vue

```diff
@@ -3,7 +3,7 @@
     <h5 slot="heading-left">Application tokens</h5>
 
     <div slot="heading-right" v-if="canCreate">
-      <toggle-link text="Create new token" :state="state" state-key="newFormVisible" class="toggle-link-new-app-token"></toggle-link>
+      <toggle-link text="Create" :state="state" state-key="newFormVisible" class="toggle-link-new-app-token"></toggle-link>
```

Comment:
So, what's the logic of changing "Create" to "Save" in some forms, but keeping "Create", for example, on this one? :confused:

Reply:
That's the toggle button and not the form submit one.